htmlToElement.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
-import {StyleSheet, Text} from 'react-native';
+import {StyleSheet, Text, Dimensions, View} from 'react-native';
 import htmlparser from 'htmlparser2-without-node-native';
 import entities from 'entities';
 
 import AutoSizedImage from './AutoSizedImage';
+import colors from '../../app/style/color.style';
 
 const defaultOpts = {
   lineBreak: '\n',
-  paragraphBreak: '\n\n',
+  paragraphBreak: '\n',
```

Comment:
I agree the paragraph break stuff is tricky, but again it's not really what you described in the PR title :smile: ... try to keep the changes constrained to what you're trying to fix.